addons/crm/models/crm_lead.py

```diff
@@ -161,6 +161,10 @@ class CrmLead(models.Model):
     date_conversion = fields.Datetime('Conversion Date', readonly=True)
     date_deadline = fields.Date('Expected Closing', help="Estimate of the date on which the opportunity will be won.")
     # Customer / contact
+    commercial_partner_id = fields.Many2one(
+        'res.partner', string='Customer Company', domain="[('is_company', '=', True)]",
+        compute="_compute_commercial_partner_id", readonly=False, store=False,
```

Comment:
Aren't computed fields non-stored by default?

```suggestion
        compute="_compute_commercial_partner_id", readonly=False,
```

Reply:
Yes but as we're marking it readonly=False we think it's better to be explicit about it.